plugin/evm/vm.go

```diff
@@ -493,7 +493,8 @@ func (vm *VM) Initialize(
 		if err != nil {
 			return errInvalidOffChainWarpMessage
 		}
-		vm.warpBackend.PutMessage(unsignedMessage)
+		vm.warpBackend.AddOffChainMessage(unsignedMessage)
+		log.Info("off-chain warp message added successfully")
```

Comment:
could we log the message as hex or a hash of it? (alternatively we could remove this log and make the log in `AddOffChainMessage` info)

Reply:
Let's move this log into the warp backend and specify the full bytes as hex in addition to the unsigned messageID so that the off chain messages are easily observable